CONVENTIONS.md

```diff
@@ -290,6 +290,24 @@ Resource request review history:
 * [BZ 1920159](https://bugzilla.redhat.com/show_bug.cgi?id=1920159) --
   for tracking changes in 4.7/4.8 for single-node RAN
 
+##### Allowed use of limits
+
+There is an exception process for adding limits to payload workloads. In the following
+scenarios workloads may be given permission to set limits:
+
+* Workloads that are scale-invariant and have a fixed memory or CPU cost no matter what the scale of the cluster or workload
+  * The workload must be demonstrated to use fixed memory or CPU, and the component must have a plan to detect whether the limit causes impact to end user workloads
+  * Workloads that have a dynamic range of usage are not a good candidate for fixed limits.
+  * Memory limits must be set to 25-50% above the observed maximum in standard e2e runs to ensure additional headroom
```

Comment:
Is this a condition for granting an exception, or something that needs to happen after the exception is granted?

Are there similar guidelines for CPU limits?

Reply:
This is probably just addition to the advice up above about how to pick your limit, so it could go up there with a pointer back to "limits need a problem"

Reply:
It seems like this new section breaks down into a couple of points

* You mostly don't want to use limits, but sometimes you do. (An intro that reinforces what was said in the parent section.)
* If you think you want limits, here are the criteria that have to be met for them to be allowed. (The 2 items on lines 298-299. Are there more?)
* After those criteria are met, here's how you pick the limits safely. (Line 301 covers it for memory, do we have similar advice for other limits?)
* After you have limits, here's the process for having them approved. (The stuff currently on line 308.)

So, I think you want this here, and maybe want to expand it, but it should be its own paragraph instead of a part of the list that it's currently in.